cmd/containerd-stargz-grpc/main.go

```diff
@@ -116,6 +121,14 @@ func main() {
 			log.G(ctx).WithError(err).Fatalf("error on serving via socket %q", *address)
 		}
 	}()
+
+	// Expose metrics
+	go func() {
+		if err := http.ListenAndServe(*metricsAddress, promhttp.Handler()); err != nil {
+			log.G(ctx).WithError(err).Fatalf("error on serving via http %q", *address)
+		}
+	}()
+
```

Comment:
We should make it optional.

Reply:
OK.

Reply:
This will introduce a bunch of go mods, too messy to manage, deleted.